13/umbraco-engage/tutorials/set-up-your-first-ab-test.md

```diff
@@ -27,7 +27,7 @@ The first step to setting up an A/B test on your website is to define a solid pl
 This tutorial does not cover defining a plan or what makes a project valid for testing. However, find some general recommendations below.
 
 * **Set a Clear Objective:** Start by defining a specific goal for your A/B test. Knowing what you aim to improve or measure allows you to design the test accordingly. For example, do you want more clicks on a button, a higher conversion rate, or something else?
-* **Ensure a valid amount of data:** A significant amount of traffic and conversions is important to produce reliable results from A/B tests. Use a tool like the Risk, Optimization, Automation, Re-think (ROAR) model to determine whether your website is eligible.
+* **Ensure a valid amount of data:** A significant amount of traffic and conversions is important to produce reliable results from A/B tests. Use a tool like the Risk Optimization Automation Re-think (ROAR) model to determine whether your website is eligible.
```

Comment:
Hm! This one it still complaining 😅 
Perhaps try without the hyphen in "Rethink"? 🤔

Reply:
Yup, that works 🤩🌟